frontend/public/components/cron-job.tsx

```diff
@@ -77,12 +79,15 @@ const CronJobTableRow: React.FC<RowFunctionArgs<CronJobKind>> = ({ obj: cronjob
       </TableData>
       <TableData className={tableColumnClasses[2]}>{cronjob.spec.schedule}</TableData>
       <TableData className={tableColumnClasses[3]}>
-        {_.get(cronjob.spec, 'concurrencyPolicy', '-')}
+        {cronjob.spec?.suspend ? i18next.t('public~true') : i18next.t('public~false')}
```

Comment:
Use `True` and `False` here and below since the CLI output uses capitalization of these values?

Reply:
@rhamilto thanks! updated